storage/src/robocorp/storage/__init__.py

```diff
@@ -42,13 +42,36 @@ def _get_client() -> "AssetsClient":
     return AssetsClient(workspace, endpoint, token)
 
 
-def list_assets() -> List[str]:
+def list_assets(page_limit: Optional[int] = 100) -> List[str]:
```

Comment:
Is this something we'd want to still retrieve all the existing assets, or do we want to retrieve the next `page_limit` ones for every subsequent manual call?

Example:
1. `list_assets(5)` -> first 5 items
2. `list_assets(10)` -> next 10 items

If yes, we can keep a context of the upcoming `next_page`, although quite useless and non-testable since the API isn't ready yet.

Reply:
I was about to comment that this is a strange. If this will return all assets, then what's the point of the page_limit?

Reply:
Without it you have no control over how many assets will be returned by the API, nor you can increase such a limit. Also it's still not clear to me how the API will [respond](https://robocorptechnologies.slack.com/archives/C053DV6NDPA/p1699004750884589?thread_ts=1699001644.429899&cid=C053DV6NDPA) in the future. (cc @bendersej )

Reply:
I think exposing the pagination could be left entirely out of this version, and rethink when it becomes an actual problem or the API is properly implemented. And even then, I'd say that should be part of the low-level client libraries you're working on and this should be more high level.